plugins/carbon_black_response/icon_carbon_black_response/actions/list_sensors/action.py

```diff
@@ -19,26 +19,24 @@ def run(self, params={}):
             ("ip", params.get("ip", "")),
             ("groupid", params.get("groupid", "")),
         ]
-        id = params.get("id", "")
+        sensorId = params.get("id", "")
```

Comment:
please use snake case instead of camel case for local variables.
```suggestion
        sensor_id = params.get("id", "")
```